### ISSUE TITLE
Tracker of etingof/pysnmp opened issues

### ISSUE THREAD

**Issue by lextm** [lextudio/pysnmp#4]
| Issue | Status | Comments |
| ----- | ------- | ---------- |
| etingof/pysnmp/issues/432 | Pending on user | Explanation was provided. |
| etingof/pysnmp/issues/431 | Pending on user | Explanation was provided. |
| etingof/pysnmp/issues/428 | Pending on user | Explanation was provided. Related to etingof/pysnmp/issues/324. |
| etingof/pysnmp/issues/424 | Pending on user | Explanation was provided. |
| etingof/pysnmp/issues/419 | TODO | |
| etingof/pysnmp/issues/418 | Pending on user | Explanation was provided. |
| etingof/pysnmp/issues/417 | TODO | |
| etingof/pysnmp/issues/411 | TODO | |
| etingof/pysnmp/issues/410 | TODO | |
| etingof/pysnmp/issues/408 | TODO | |
| etingof/pysnmp/issues/407 | TODO | |
| etingof/pysnmp/issues/406 | TODO | |
| etingof/pysnmp/issues/405 | TODO | |
| etingof/pysnmp/issues/404 | TODO | |
| etingof/pysnmp/issues/402 | TODO | |
| etingof/pysnmp/issues/401 | TODO | |
| etingof/pysnmp/issues/400 | TODO | |
| etingof/pysnmp/issues/399 | TODO | |
| etingof/pysnmp/issues/396 | TODO | |
| etingof/pysnmp/issues/393 | TODO | |
| etingof/pysnmp/issues/392 | TODO | |
| etingof/pysnmp/issues/391 | TODO | |
| etingof/pysnmp/issues/387 | TODO | |
| etingof/pysnmp/issues/384 | | Linked to etingof/pyasn1/pull/215. |
| etingof/pysnmp/issues/382 | TODO | |
| etingof/pysnmp/issues/381 | TODO | |
| etingof/pysnmp/issues/378 | TODO | |
| etingof/pysnmp/issues/377 | TODO | |
| etingof/pysnmp/issues/371 | TODO | |
| etingof/pysnmp/issues/370 | TODO | |
| etingof/pysnmp/issues/369 | TODO | |
| etingof/pysnmp/issues/368 | TODO | A good feature request. |
| etingof/pysnmp/issues/367 | TODO | |
| etingof/pysnmp/issues/366 | TODO | |
| etingof/pysnmp/issues/363 | TODO | | 
| etingof/pysnmp/issues/361 | TODO | |
| etingof/pysnmp/issues/360 | TODO | |
| etingof/pysnmp/issues/358 | TODO | A doc issue. |
| etingof/pysnmp/issues/357 |  | Linked to etingof/pyasn1/pull/215. |
| etingof/pysnmp/issues/356 | TODO | |
| etingof/pysnmp/issues/355 | TODO | |
| etingof/pysnmp/issues/353 | TODO | |
| etingof/pysnmp/issues/352 | TODO | |
| etingof/pysnmp/issues/343 | TODO | Python 3.8 incompatibility. |
| etingof/pysnmp/issues/342 | TODO | |
| etingof/pysnmp/issues/341 | TODO | |
| etingof/pysnmp/issues/336 | TODO | |
| etingof/pysnmp/issues/335 | TODO | |
| etingof/pysnmp/issues/331 | TODO | |
| etingof/pysnmp/issues/330 | TODO | |
| etingof/pysnmp/issues/329 | TODO | |
| etingof/pysnmp/issues/328 | TODO | |
| etingof/pysnmp/issues/327 | TODO | |
| etingof/pysnmp/issues/316 | TODO | |
| etingof/pysnmp/issues/315 | TODO | |
| etingof/pysnmp/issues/307 | TODO | |
| etingof/pysnmp/issues/306 | TODO | |
| etingof/pysnmp/issues/305 | TODO | |
| etingof/pysnmp/issues/290 | TODO | |
| etingof/pysnmp/issues/283 | TODO | |
| etingof/pysnmp/issues/281 | TODO | |
| etingof/pysnmp/issues/280 | TODO | |
| etingof/pysnmp/issues/274 | TODO | |
| etingof/pysnmp/issues/273 | TODO | |
| etingof/pysnmp/issues/270 | TODO | |
| etingof/pysnmp/issues/268 | TODO | |
| etingof/pysnmp/issues/266 | TODO | |
| etingof/pysnmp/issues/265 | TODO | |
| etingof/pysnmp/issues/264 | TODO | |
| etingof/pysnmp/issues/261 | TODO | |
| etingof/pysnmp/issues/257 | TODO | |
| etingof/pysnmp/issues/255 | TODO | |
| etingof/pysnmp/issues/254 | TODO | |
| etingof/pysnmp/issues/251 | TODO | A doc issue. |
| etingof/pysnmp/issues/250 | TODO | |
| etingof/pysnmp/issues/237 | TODO | pyasn1 issue. |

Didn't check further <237, as the current backlog is already too long.

Old 5.0 roadmap is at https://github.com/etingof/pysnmp/projects/1

**Comment by lextm** [comment]
Opened pull requests

* etingof/pysnmp/pull/226 (TCP support): Superseded by v7.1.28 implementation.

**Comment by lextm** [comment]
Important changes not cherry-picked yet,

* d7534580837445553d8ffdc86fd5a161b9d57af1, cherry picked and improved in v7.2.
* 039b4e5204431b2b1eee3c2241278516404fc2ef, cherry picked.
* f31d8923d5343c167448738d0803407e3272c198, already included.
* b9a7b9c955df98b1e5eec250f418c3df3b7d0199, improvements of examples but superseded.
